File: src/main/java/com/github/marciel404/bot/Main.java
package com.github.marciel404.bot;

import net.dv8tion.jda.api.JDA;
import com.github.marciel404.bot.utils.LoaderCommands;

import static com.github.marciel404.bot.utils.loader.client;

public class Main {
    public static void main(String[] args){
        //System.setProperty(org.slf4j.simple.SimpleLogger.LOG_FILE_KEY, "null");
        try{
            JDA client1 = client;
            LoaderCommands.load();
        } catch (Exception error){
         System.out.println("Error: " + error);
        }
    }
}

File: src/main/java/com/github/marciel404/bot/Main.kt
package com.github.marciel404.bot

import com.github.marciel404.bot.utils.Client
import com.github.marciel404.bot.utils.loadCommands
import org.slf4j.simple.SimpleLogger

object Main {
    @JvmStatic
    fun main(args: Array<String>) {
        System.setProperty(SimpleLogger.LOG_FILE_KEY, "logs")
        try {
            Client.client
            loadCommands()
        } catch (error: Exception) {
            println("Error: $error")
        }
    }
}